great_expectations/rule_based_profiler/types/data_assistant_result/volume_data_assistant_result.py

```diff
@@ -92,7 +96,7 @@ def plot(
 
         charts.append(table_row_count_chart)
 
-        self.display(charts=charts)
+        self.display(charts=charts, custom_altair_theme=custom_altair_theme)
```

Comment:
Ideally our default theme will cover everything, so that we don't need to extend it at the assistant level. This will also help in "re-theming" all of our plots, because the default theme will define everything that is known

Reply:
... and then we don't have to pass in this parameter

Reply:
This parameter is the user specified theme, so I don't see a way around passing it. Unless I'm missing something.

Reply:
Since the `AltairThemes.DEFAULT_THEME` should include whatever we need, so we don't need to pass anything here.

Reply:
So we no longer want the user to be able to pass a theme at runtime? That is what is being passed here.

Reply:
ah, now I get it -- thanks!